pkg/cgroup/resolve_container.go

```diff
@@ -52,7 +52,7 @@ var (
 	podLister kubelet.KubeletPodLister = kubelet.KubeletPodLister{}
 
 	// map to cache data to speedup lookups
-	containerIDCache           = map[uint64]string{}
+	ContainerIDCache           = map[uint64]string{}
```

Comment:
If the map is accessible outside of the pkg, it could be misused and potentially cause race condition. If you need to get the map directly, create a utility function that returns a copy of it.

Reply:
Makes sense, I'm removing this.